src/test/features/response/routes/summary.ts

```diff
@@ -0,0 +1,61 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from '../../../routes/hooks'
+import { checkAuthorizationGuards } from './checks/authorization-check'
+
+import { Paths as ResponsePaths } from 'response/paths'
+
+import { app } from '../../../../main/app'
+
+import * as idamServiceMock from '../../../http-mocks/idam'
+import * as claimStoreServiceMock from '../../../http-mocks/claim-store'
+
+import { checkNotDefendantInCaseGuard } from './checks/not-defendant-in-case-check'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const pagePath: string = ResponsePaths.summaryPage.evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId })
+
+describe('Defendant response: summary page', () => {
+  attachDefaultHooks(app)
+
+  describe('on GET', () => {
+    const method = 'get'
+    checkAuthorizationGuards(app, method, pagePath)
+    checkNotDefendantInCaseGuard(app, method, pagePath)
```

Comment:
This guard is really checking that only parties should have access. We could rename it laster to something like `checkPartyAccessGuards` or `checkOnlyAllowedPartiesHaveAccess`